frontendservice/src/views/specificShop.vue

```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
+        self.getWaitingTime();
+
+        if(this.isTokenInCookie) {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.tokeninfo.customersAhead;
+        } else{
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.shopinfo.customersInQueue;
+        }
+    },
+
```

Comment:
remove extra line.. here and at other places

Reply:
Done.